### PR TITLE
Default Value Struct changes

### PR DESCRIPTION
**Pull request by NirnayaSindhuSuthari**
- Added DefaultValue field of DefaultValue type in Column struct.
- Added DefaultValue struct.

### CODE REVIEW

File: sources/mysql/infoschema_test.go [hunk 1]
```diff
@@ -389,13 +400,14 @@ func TestProcessData_MultiCol(t *testing.T) {
 				"a":        ddl.ColumnDef{Name: "a", T: ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, NotNull: true},
 				"b":        ddl.ColumnDef{Name: "b", T: ddl.Type{Name: ddl.Float64}},
 				"c":        ddl.ColumnDef{Name: "c", T: ddl.Type{Name: ddl.Int64}},
+				// "d":		ddl.ColumnDef{Name: "d", T: ddl.Type{Name: ddl.Int64}, DefaultValue: },
 				"synth_id": ddl.ColumnDef{Name: "synth_id", T: ddl.Type{Name: ddl.String, Len: 50}},
```
Comment (asthamohta) [inline sources/mysql/infoschema_test.go:404]: Fix this too

File: sources/mysql/infoschema_test.go [hunk 2]
```diff
@@ -361,6 +371,7 @@ func TestProcessData_MultiCol(t *testing.T) {
 				{"a", "text", "text", "NO", nil, nil, nil, nil, nil},
 				{"b", "double", "double", "YES", nil, nil, 53, nil, nil},
 				{"c", "bigint", "bigint", "YES", nil, nil, 64, 0, nil}},
+				// {"d", "varchar", "varchar", "YES", "John", nil, nil, nil, nil}},
```
[the remark this answers is not in the document]
Comment (NirnayaSindhuSuthari) [reply sources/mysql/infoschema_test.go]: done.

File: sources/mysql/infoschema_test.go (hunk 1 above)
Comment (NirnayaSindhuSuthari) [reply sources/mysql/infoschema_test.go]: done.

File: sources/mysql/infoschema_test.go [hunk 3]
```diff
@@ -395,7 +405,7 @@ func TestProcessData_MultiCol(t *testing.T) {
 	}
 	internal.AssertSpSchema(conv, t, expectedSchema, stripSchemaComments(conv.SpSchema))
 	columnLevelIssues := map[string][]internal.SchemaIssue{
-		"c49": []internal.SchemaIssue{
+		"c5": []internal.SchemaIssue{
```
Comment (darshan-sj) [inline sources/mysql/infoschema_test.go:408]: Why is this change required? `c49` to `c5` ?

**Review by darshan-sj — COMMENTED** [review]
Changes look good. Please check the github check failures and fix them.

File: sources/mysql/infoschema_test.go (hunk 3 above)
Comment (NirnayaSindhuSuthari) [reply sources/mysql/infoschema_test.go:408]: since few columns are added, there is a change in column from c49 to c5.
